src/erocksdb.erl

```diff
@@ -461,6 +462,12 @@ repair(_Name, _DBOpts) ->
     erlang:nif_error({error, not_loaded}).
 
 
+%% @doc for each stores retun the approximate size in a range
+-spec get_approximate_size(db_handle(), binary(), binary(), boolean()) -> integer().
+get_approximate_size(_DbHandle, _SKey, _EKey, _IncludeMemtable) ->
+  erlang:nif_error({error, not_loaded}).
+
```

Comment:
Please add the unit tests as follows.
1. check the return value whether it's `0` when there is no records between StartKey and EndKey.
2. check the return value whether it's `> 0` when there are some records between StartKey and EndKey.